core/src/main/java/tc/oc/pgm/filters/modifier/QueryModifier.java

```diff
@@ -42,4 +45,14 @@ protected QueryResponse queryTyped(Q query) {
 
     return modifiedQuery == null ? QueryResponse.ABSTAIN : filter.query(modifiedQuery);
   }
+
+  /**
+   * ONLY VISIBLE FOR {@link Filters#flattenFilter(Filter, List)}
+   *
+   * @return the child filter
+   */
+  @ApiStatus.Internal
```

Comment:
what's the need to be so restrictive, its' fine if this is visible imo

Reply:
The thought was to restrict access such that no-one thinks that this method is public API that returns a Filter that would respond in the same way as this modifier with a set child.

Reply:
Consider just naming this `getChild` or `getChildFilter` instead, avoids the confusion of people thinking it's directly the same filter

Reply:
This method is not necessary because all filters can be found the feature definition context anyways